godotproject/PlayerTongue.gd

```diff
@@ -2,8 +2,8 @@ extends Node2D
 
 signal tongue_swing
 
-export (float) var shoot_speed = 600
-export (float) var max_shoot_dist = 250
+export (float) var shoot_speed = 100000
```

Comment:
```suggestion
export (float) var shoot_speed = 1500
```
This should be a more reasonable shoot speed. To correct the project to use the shoot_speed in this file, you need to open Player.tscn in the editor, select the PlayerTongue scene, and hit the reset button beside the Shoot Speed in the inspector on the right. Can you make another commit into this branch with these changes? (feel free to try out other shoot speeds than 1500)

Reply:
for sure!